src/libpmemfile-core/data.c

```diff
@@ -120,11 +119,13 @@ is_block_data_initialized(const struct pmemfile_block *block)
 }
 
 /*
- * find_block -- look up block metadata with the highest offset
+ * find_block_cach -- look up block metadata with the highest offset
  * lower than or equal to the offset argument
+ *
+ * using the block_pointer_cache field in struct pmemfile_file
  */
 static struct pmemfile_block *
-find_block(struct pmemfile_file *file, uint64_t offset)
+find_block_cach(struct pmemfile_file *file, uint64_t offset)
```

Comment:
Typo.

Maybe leave this function as is and rename the new one to find_block_uncached?